DEVDOCS-4635: [new] Users, Account-level API accounts (#1841)
bigcommerce/dev-docs: docs/api-docs/users/Account-level-API-accounts.mdx

# Account-level API accounts

## Introduction

> Account-level API tokens enable store owners to manage account-level permissions per BigCommerce account. When generated, this newly created token will belong to the account that store owner has logged into. 

Reviewer: it is called Account API tokens (based on discussion with Philip Christian and the UX team)

# Account-level API accounts

## Introduction
The Account-Level API is designed to allow merchants to build custom integrations and solutions that require high-level access to a store's resources. By using this API, merchants can automate various business need processes, integrate with third-party applications, and manage multiple stores per account.

<Callout type="info">

>     When a token is generated it will belong to the account that the merchant has logged into.

Reviewer 2: ```suggestion
    When a token is generated, it will belong to the account that the merchant has logged into.
```

</Callout> 

With Account-Level API credentials merchants can now perform actions such as:
* Create tokens that can be used to manage apps on a merchant's store
* Manage charges and subscriptions for accounts
* Integrate third-party applications to streamline and automate store processes

<Callout type="info">
    As long as this account is active, the token will always be tied to the account.
</Callout>

Before you get started creating Account-Level tokens, read our doc on [API Accounts & OAuth Scopes](api-docs/getting-started/api-accounts) to get up to speed.

## Prerequisites

Head into action with Account-level API management by verifying that you have the following prerequisites.

* A BigCommerce store
* A valid API account access token with the necessary credentials to make these changes

> ## OAuth scopes

Reviewer 2: @bc-tgomez I believe these need to be added to the [OAuth scopes article](https://developer.bigcommerce.com/api-docs/getting-started/api-accounts#oauth-scopes)?

Author: Here's the [new ticket](https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4868) for reference.

Be sure to set the following OAuth scopes to acquire the permissions needed to manage the Account-Level API:

| Name | Permission | Description |
|:---|:---|:---|
| Subscriptions | read-only | View or cancel subscriptions |
| Checkouts | modify | View or cancel checkouts |
| Charges | modify | Create or delete charges |
| Account | read-only | View account information for Account level GraphQL |
| Account Stores | read-only | View information for apps belonging to the account for the Account-level GraphQL |
| Accounts Apps | read-only | View information for apps belonging to the account for Account-Level API |
| Account User | modify | View/Write/Delete information for users belonging to the account for Account-Level GraphQL |

## How it works

> To use the Account-Level API, merchants need to generate an API token from the BigCommerce control panel. The API token should be kept secure, as it provides complete access to the account's stores. This means that any actions performed through the API have the same effect as if they were performed directly in the store's control panel. 

Reviewer 3: ```suggestion
To use the Account-Level API, merchants must generate an API token from the BigCommerce control panel. The API token should be kept secure, as it provides complete access to the account's stores. This means that any actions performed through the API have the same effect as if they were performed directly in the store's control panel. 
```

> Once an API call is made to the BigCommerce API gateway, the gateway will then check for the proper token permissions. If the token has the correct rights, the API request can be made. However, if the token does not have the correct permission rights, the API request will bounce.

Reviewer 3: ```suggestion
Once an API call is made to the BigCommerce API gateway, the gateway will check for the proper token permissions. If the token has the correct rights, the API request can be made. However, if the token does not have the correct permission rights, the API request will bounce.
```

> This process allows BigCommerce to filter out API requests that aren't verified.

Reviewer 3: ```suggestion
This process allows BigCommerce to filter out API requests that must be verified.
```

> All in all, the Account-Level API accounts on the BigCommerce platform provides merchants and developers with a powerful tool-set for building custom integrations and solutions that can help streamline and automate various processes.

Reviewer 3: ```suggestion
All in all, the Account-Level API accounts on the BigCommerce platform provide merchants and developers with a powerful tool-set for building custom integrations and solutions that can help streamline and automate various processes.
```
